nyx_client/utils.py

```diff
@@ -176,6 +176,7 @@ def data_as_db(
             return db_engine
         tables = []
         for d in data:
+            table_name = Parser.normalise_values([d.title])[0]
```

Comment:
should likely make that function more robust and replace any special chars with underscores